src/commands/mod.rs

```diff
@@ -622,6 +635,12 @@ impl DaemonControl {
         if feerate_vb < 1 {
             return Err(CommandError::InvalidFeerate(feerate_vb));
         }
+        if address.network != self.config.bitcoin_config.network {
```

Comment:
beware that address network can be testnet, and bitcoin_config.network signet

Reply:
Ah good catch! I had a note to double check that and rushed and forgot about it. Thanks.